config/locales/doorkeeper.zh-CN.yml

```diff
@@ -30,60 +31,65 @@ zh-CN:
       form:
         error: 抱歉! 提交信息的时候遇到了下面的错误
       help:
-        native_redirect_uri: 使用 %{native_redirect_uri} 作为本地测试
+        native_redirect_uri: 本地测试请使用 %{native_redirect_uri}
         redirect_uri: 每行只能有一个 URL
-        scopes: 用空格隔开权限范围，留空则使用默认设置
+        scopes: 用空格分割权限范围，留空则使用默认设置
       index:
-        callback_url: 登录回调地址
+        application: 应用
+        callback_url: 回调 URL
+        delete: 删除
         name: 名称
         new: 创建新应用
+        scopes: 权限范围
+        show: 显示
         title: 你的应用
       new:
         title: 创建新应用
       show:
         actions: 操作
-        application_id: 应用 ID
-        callback_urls: 登录回调地址
+        application_id: 客户端 ID
```

Comment:
Not a good idea to turn AppID into client ID here. Also should we send this back upstream too? See https://github.com/tootsuite/documentation/blob/master/Contributing-to-Mastodon/Translating.md#translating-1 for a list of upstream repos.

Reply:
@Artoria2e5 I'll open a pull request to the upstream repos after we merge this